packages/orbit-components/src/SmartPassIllustration/components/SmartPassV2.js

```diff
@@ -0,0 +1,272 @@
+// @flow
+
+import React from "react";
+
+import type { Props } from "../index";
+
+type SvgProps = {|
+  ...Props,
+  height: string,
+|};
+
+const SmartPassV2 = ({
+  primary,
+  secondary,
+  height,
+  name,
+  title,
+  description,
+  ariaLabelledby,
+  dataTest,
+}: SvgProps) => {
+  return (
+    <svg
+      height={height}
+      data-test={dataTest}
+      viewBox="0 0 2200 1300"
+      fill="none"
+      role="img"
+      aria-labelledby={ariaLabelledby}
```

Comment:
I don't know what passing `ariaLabelledby` to `SmartPassIllustration` and forwarding it to SVG elements does, but I assume you researched it and know that it's the right thing to do. 🤷‍♂️